packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2691,8 +2692,8 @@ export interface RemoteVideoStreamState {
     view?: VideoStreamRendererViewState;
 }
 
-// @beta (undocumented)
-export type Role = 'Presenter' | 'Attendee' | 'Consumer';
+// @beta
+export type Role = 'Unknown' | 'Presenter' | 'Attendee' | 'Consumer' | 'Organizer';
```

Comment:
Is unknown from the Calling SDK? I would suggest role being `'Presenter' | 'Attendee' | 'Consumer' | 'Organizer'` only and then it should be passed as undefined it is unknown or not used.
Or is there an SDK role type we can use?

Reply:
From the meeting this morning, as things move to the selector layer and adpater ctor this interface may be removed as well so not blocking on this comment

Reply:
Yes, 'Unknown' is from from the Calling SDK also.

Reply:
Just a note that `react-components` should only list the values they care about, not necessarily reflect the Calling SDK exactly. What is `unknown` to a react-component? When would Contoso, when using a pure component, pass `unknown` instead of undefined?